### PR TITLE
Fixes #19276: Remove reference to x-www-form-urlencoded in API commands

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/19276

### CODE REVIEW

File: webapp/sources/api-doc/introduction.md [hunk 1]
```diff
@@ -263,3 +171,100 @@ They must be part of the query and can't be submitted in a JSON form.
...
 </table>
+
+
+### Passing parameters
+
+Parameters to the API can be sent:
+
+* As part of the URL for resource identification
+
+* Sending data for POST/PUT requests
```
Comment (amousset) [inline webapp/sources/api-doc/introduction.md:182]:
```suggestion
* As data for POST/PUT requests
```

File: webapp/sources/api-doc/introduction.md [hunk 2]
```diff
@@ -263,3 +171,100 @@ They must be part of the query and can't be submitted in a JSON form.
...
+The supplied file must contain a valid JSON: strings need quotes, booleans and integers don't, etc.
+
+The (human readable) format is:
+
+
+    {
+      "key1": "value1",
+      "key2": false,
+      "key3": 42
+    }
```
Comment (amousset) [inline webapp/sources/api-doc/introduction.md:216]: <pre>
```json
{
  "key1": "value1",
  "key2": false,
  "key3": 42
}
```
</pre>
should allow proper coloration

**Comment by fanf** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**